src/connectionmonitor/SignalingAndMetricsConnectionMonitor.ts

```diff
@@ -186,26 +186,32 @@ export default class SignalingAndMetricsConnectionMonitor
       return;
     }
 
-    const isHardwareEncoder = clientMetricReport.getObservableVideoMetricValue(
-      'videoUpstreamEncoderImplementation',
-      ssrc
-    );
-    const videoEncodingTimeInMs = clientMetricReport.getObservableVideoMetricValue(
-      'videoUpstreamTotalEncodeTimePerSecond',
-      ssrc
-    );
-    const cpuLimitationDuration = clientMetricReport.getObservableVideoMetricValue(
-      'videoUpstreamCpuQualityLimitationDurationPerSecond',
-      ssrc
-    );
-    const videoInputFps = clientMetricReport.getObservableVideoMetricValue(
-      'videoUpstreamFramesInputPerSecond',
-      ssrc
-    );
-    const videoEncodeFps = clientMetricReport.getObservableVideoMetricValue(
-      'videoUpstreamFramesEncodedPerSecond',
-      ssrc
-    );
+    let isHardwareEncoder = false;
+    let videoEncodingTimeInMs = 0;
+    let cpuLimitationDuration = 0;
+    let videoInputFps = 0;
+    let videoEncodeFps = 0;
+    for (const ssrc of ssrcs) {
```

Comment:
Could use a comment explaining this. How does it impact how we should interpret the metrics?

Reply:
Added a comment to explain aggregation of the metrics.